www/base/src/styles/styles.less

```diff
@@ -74,6 +74,9 @@
   }
 }
 
+.label {
```

Comment:
I merged to be able to ship it in 0.9.10. I will see how it works from the release packages, and see if I ship it like this.
I would like indeed a followup to cleanup the code